src/args/__tests__/parse-args.test.ts

```diff
@@ -12,10 +12,13 @@ Deno.test("Should parse --deno-log-level flag", () => {
     "--deno-log-level",
     "debug",
     "--deno-keep-tmp-chart",
+    "--deno-importmap",
```

Comment:
Let's separate words with dash for better readability
```suggestion
    "--deno-import-map",
```

Reply:
Yes, this name of the option is better